src/CoreGraphics/CGFont.cs

```diff
@@ -139,20 +139,12 @@ public static CGFont CreateWithFontName (string name)
 			}
 		}
 
-		string GetNullableString (IntPtr handle)
-		{
-			if (handle == IntPtr.Zero)
-				return null;
-			using (var s = new CFString (handle, true))
-				return (string) s;
-		}
-
 		[DllImport (Constants.CoreGraphicsLibrary)]
 		extern static /* CFStringRef __nullable */ IntPtr CGFontCopyPostScriptName (/* CGFontRef __nullable */ IntPtr font);
 
 		public string PostScriptName {
 			get {
-				return GetNullableString (CGFontCopyPostScriptName (handle));
+				return CFString.FetchString (CGFontCopyPostScriptName (handle));
```

Comment:
I think this will leak, because the CFString was previously created with `owns=true`.